blog-posts/1-meet-the-hoodie-@espy.md

```diff
@@ -0,0 +1,45 @@
+
+The best thing was that Hoodie saved me so much work implementing the authentication, data storage and sync that I could spend much more time on the UI and the user experience, which really paid off.
+
+###*What in your opinion makes the Hoodie project unique and interesting?*
+
+The people-centered approach to, well, basically everything. It starts with how we desigend the API: we didn‘t go down the tech-centered path you see so often in Open Source, simply exposing server-side features to the front-end, but focusing instead on what developers would need and want. User-centered API design.
+
+This approach extends to how we do work in general, trying to accomodate people and their needs as well as we can, and being open, inclusive and accessible, also valuing all sorts of work highly, not just coding. This interview, for example, is the product of our new Open Source editorial team, an attempt at finding contributors and contributions for Hoodie’s outward communication.
+
+We’ve often been told that people are attracted to how we do things as much as to the things that we do, and we feel very validated by this. Open Source is more than just code. Code changes and becomes deprecated, but culture and people persist far beyond that.
+
+I like that we value independence and autonomy highly, in that there's no VC funding, the project is completely Open Source, can’t be sold or acquired, and is generally a safe bet: it’s not going to vanish overnight, you’re not going to lose your data and your storage platform because we got bought, and we’re not going to drop the whole thing once a lucrative exit presents itself. Hoodie is our exit strategy. We want this to be A Thing™.
+
+###*What advice do you have for aspiring Hoodie contributors?*
+
+First off, don't be shy, we’re pretty easy to get in touch with, both over twitter @hoodiehq, as well as our combined Slack/IRC channel (find out more on our chat page). We’ve also had people just show up out of nowhere with a pull request on GitHub, that’s also cool. But in general, just come talk to us!
+
+As for what you can do: we’ve built a tool called ubersicht to help you (and us, if we're honest) find issues all across our organisation, and there’s a big green button to display starter issues for new or beginner contributors.
+
+On the tech side, we use CouchDB/PouchDB, node.js, and Hapi for the internals. We’re also super happy when people test and write tutorials, keep the docs in shape, write posts and articles, make code and design improvements to the websites or just try out Hoodie itself and give us feedback on it.
+
+So there’s a wide range of topics and difficulties to pick from, and a bunch of friendly people to talk to about the whole thing, too. Jump in!
+
+And finally, here's me, grinning like an idiot on my first Shinkansen ride:
+
+![Grinning Like an idiot on my first Shinkansen ride](https://cloud.githubusercontent.com/assets/391124/11560507/567f3f1c-99c1-11e5-8454-a4a7497c06aa.jpg)
```

Comment:
"Idiot" is definetly a word we don't use at Hoodie, can we find an alternative please? "Grinning on my first Shinkansen" ride would work. Both for the title of the image, and in the explanative text before hand.